Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -50,41 +52,46 @@ u64 WbfsFileReader::GetDataSize() const
   return WII_SECTOR_COUNT * WII_SECTOR_SIZE;
 }
 
-bool WbfsFileReader::OpenFiles(const std::string& filename)
+void WbfsFileReader::OpenAdditionalFiles(const std::string& path)
 {
-  m_total_files = 0;
+  if (path.length() < 4)
+    return;
 
   while (true)
   {
-    auto new_entry = std::make_unique<file_entry>();
-
     // Replace last character with index (e.g. wbfs = wbf1)
-    std::string path = filename;
-    if (m_total_files != 0)
-    {
-      path[path.length() - 1] = '0' + m_total_files;
-    }
+    std::string current_path = path;
+    if (m_files.size() >= 10)
```

Comment:
Ah, I see, I was just thinking of how to optimize the loop, forgot you're opening a file every time.  Would it make more sense to check the path for existence first instead of opening the file?

And do you all use asserts?  Might make sense to assert that the initial file-size is 1?